editor/plugins/org.fusesource.ide.projecttemplates/template-sources/blank/blank-blueprint/pom.xml

```diff
@@ -35,7 +35,7 @@
 		<repository>
 			<id>fuse-public-repository</id>
 			<name>FuseSource Community Release Repository</name>
-			<url>https://repo.fusesource.com/nexus/content/groups/public</url>
+			<url>${fusesourceNexusCommunity}</url>
```

Comment:
these poms are not part of the build, they are templates. So unless you are replacing the value in the file during the build I don't think that it is working.